modules/common-fs2/src/main/scala/com/snowplowanalytics/snowplow/enrich/common/fs2/Environment.scala

```diff
@@ -198,7 +190,25 @@ object Environment {
       metadata <- Resource.eval(metadataReporter[F](file, processor.artifact, http))
       assets = parsedConfigs.enrichmentConfigs.flatMap(_.filesToCache)
       (remoteAdaptersHttpClient, remoteAdapters) <- prepareRemoteAdapters[F](file.remoteAdapters, ec, metrics)
-      adapterRegistry = new AdapterRegistry(remoteAdapters)
+      adapterRegistry = new AdapterRegistry(
+                          remoteAdapters,
+                          callrailConfig = file.adapters.callrail,
```

Comment:
```suggestion
      adapterRegistry = new AdapterRegistry(remoteAdapters, file.adapters)
```
It'd be better to let `AdapterRegistry` pick the configs

Reply:
Have changed this to accept a `AdaptersSchemas` instance with all the configs encapsulated.